include/hermes/options.hpp

```diff
@@ -8,6 +8,7 @@ enum __engine_opts {
     __use_auto_sm      = 1L << 0,
     __print_stats      = 1L << 1,
     __force_no_block_progress = 1L << 2,
+    __fork_save = 1L << 3,
```

Comment:
Please rename option to `process_may_fork` to make intent more explicit.

Reply:
Solved in 87b51d2a2b37dbfee7826bc9a61acc69ac293e62